context/app/static/js/components/entity-search/sidebar/DatasetConfigureSearchTable/hooks.js

```diff
@@ -58,13 +60,15 @@ function useFieldGroups() {
 }
 
 function buildFieldConfigs(fieldNames) {
-  return fieldNames.reduce(
-    (acc, fieldName) => ({
+  return fieldNames.reduce((acc, fieldName) => {
+    if (fieldName === 'version') {
```

Comment:
```suggestion
    const excludedFieldNames = ['version'];
    if (excludedFieldNames.includes(fieldName) {
```
In response to your question at the top: no others come to mind... but there probably will be more, so let's make it easier to find the code that does the work.